app/components/post_body_additional_content/post_body_additional_content.js

```diff
@@ -157,7 +166,11 @@ export default class PostBodyAdditionalContent extends PureComponent {
     };
 
     generateToggleableEmbed = (isImage, isYouTube) => {
-        const {link} = this.props;
+        let {link} = this.props;
```

Comment:
This could be simplified by doing `const link = this.state.shortenedLink || this.props.link` or there could be a method to do that so you don't need to type it out repeatedly. 0/5 though